.github/workflows/swift-test-and-build.yaml

```diff
@@ -6,13 +6,13 @@ on:
 
 jobs:
   test-and-build:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
There is a way to launch an app into [an iOS simulator](https://github.com/marketplace/actions/launch-ios-simulator) but it would still be inside of a macOS runner I believe.

Reply:
does it have to be an app? can't it run a library?